src/App.js

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-import { connect } from 'react-redux';
-
-import NavigationContainer from './nav/NavigationContainer';
-
-const App = (props) =>
-  <NavigationContainer />;
-
-const mapStateToProps = (state) => ({
-  isLoggedIn: state.app.isLoggedIn,
```

Comment:
If for some reason the apiKey stored from last time is not valid (the user changed password etc.) we need to handle this. I think I did add code about it, not sure if the feature is complete.